astropy/units/format/generic_parsetab.py

```diff
@@ -1,77 +1,76 @@
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
@mdboom - on both this comment and that on the appearance of the full path below: is there a way to automate this? I'll make the changes by hand here, but it seems error prone (and it'll have to be done again for unicode, etc.).